src/filters/oidc/oidc_filter.cc

```diff
@@ -182,26 +191,46 @@ google::rpc::Code OidcFilter::Process(
   }
    */
 
-  // Check if an authorization header already exists. If so let request
+  // Check if an id_token header already exists. If so let request
   // progress. It is up to the downstream system to validate the header is
   // valid.
   auto headers = request->attributes().request().http().headers();
-  if (headers.contains(common::http::headers::Authorization)) {
+  if (headers.contains(idp_config_.id_token().header())) {
```

Comment:
@nickrmc83 I think what you're doing here is allowing the end-user to add the right header if they, for example, manually construct a curl to the app which the authservice is protecting.  That seems to make sense as a feature.  Now that the code is handling both ID Tokens and Access Tokens, would it make sense to skip token acquisition when the end user manually adds *either* the configured ID Token header *or* the configured Access Token header?

Reply:
This is where I was looking for more input about what the expected behaviour should be. I'm happy to to skip acquisition if either are supplied.